src/gui/mainwindow.h

```diff
@@ -196,6 +197,8 @@ class MainWindow : public wxFrame, public WindowAttributesPickle
         static const unsigned int OPT_PAGE_GROUPS   = 4;
 
 	const wxArrayString& GetTabNames();
+	
+	static const unsigned int mySHOW_ERROR_MESSAGE = 10000;
```

Comment:
http://docs.wxwidgets.org/trunk/group__group__funcmacro__misc.html#gaa2d93c1680369b4227f7de9de58e5fcb
Looks like this is not a good idea